src/components/sdk/ContentShare/ContentShare.mdx

```diff
@@ -5,12 +5,16 @@ import { ContentShare } from './';
 
 # ContentShare
 
-The `ContentShare` component renders a `ContentTile` for the active content share video, remote or local.
+The `ContentShare` component renders a `ContentTile` for a content share video, remote or local.
 
 If used within the `VideoGrid` component, it will automatically place the active tile in the featured grid slot. It takes precedence over the featured video tile.
 
 Once a meeting session has been started, a user can start and stop content sharing by using the `useContentShareControls` hook.
 
+## Multiple Content Shares
+
+With the support for multiple content shares, you can now specify which content share tile to render by providing the `tileId` prop. If no `tileId` is provided, the component will render the default content share tile from (`const { tileId } = useContentShareState()`).
```

Comment:
So what will the `tileId` default to in case one does not provide it as a prop? Suggest adding a clarification for builder to help get answer to this question in the docs.

Reply:
We should also link to `ContentShareProvider` with a small explanation on how `ContentShareProvider maxContentShares={2}` impacts the `ContentShare` component.

Reply:
>If no `tileId` is provided, the component will render the default content share tile from (`const { tileId } = useContentShareState()`).

I think this is clear enough. It will use the existing `tileId` property from `useContentShareState()` hook

Reply:
>We should also link to ContentShareProvider with a small explanation on how ContentShareProvider maxContentShares={2} impacts the ContentShare component.

To clarify, `maxContentShares` doesn't impact `ContentShare`.